docs/installation/manual/installing_solr.md

```diff
@@ -0,0 +1,99 @@
+
+```bash
+cd /opt/solr
+sudo mkdir -p /var/solr/data/SOLR_CORE/conf
+sudo cp -r example/files/conf/* /var/solr/data/SOLR_CORE/conf
+sudo chown -R solr:solr /var/solr
+sudo -u solr bin/solr create -c SOLR_CORE -p 8983
+```
+- `SOLR_CORE`: `islandora8`
+
+### Installing `search_api_solr`
+
+Rather than use an out-of-the-box configuration that won’t be suitable for our purposes, we’re going to use the Drupal `search_api_solr` module to generate one for us. This will also require us to install the module so we can create these configurations using Drush.
+
+```bash
+cd /opt/drupal
+sudo -u www-data composer require drupal/search_api_solr:^3.0
+drush -y en search_api_solr
+```
+
+### Configuring search_api_solr
+
+Before we can create configurations to use with Solr, the core we created earlier needs to be referenced in Drupal.
+
+Log in to the Drupal site at `/user` using the sitewide administrator username and password, then navigate to `/admin/config/search/search-api/add-server`.
+
+Fill out the server addition form using the following options:
+
+![Adding a Solr Search Server](../../assets/adding_a_solr_search_server.png)
+
+![Configuring the Standard Solr Connector](../../assets/configuring_standard_solr_connector.png)
+
+![Setting the Solr Install Directory](../../assets/setting_the_solr_install_directory.png)
+
+- `SERVER_NAME`: `islandora8`
+    - This is completely arbitrary, and is simply used to differentiate this search server configuration from all others. **Write down** or otherwise pay attention to the `machine_name` generated next to the server name you type in; this will be used in the next step.
+
+!!! notice
+    You can ignore the error about an incompatible Solr schema; we're going to set this up in the next step. In fact, if you refresh the page after restarting Solr in the next step, you should see the error disappear.
+
+### Generating and Applying Solr Configurations
+
+Now that our core is in place and our Drupal-side configurations exist, we’re ready to generate Solr configuration files to connect this site to our search engine.
+
+```bash
+cd /opt/drupal
+drush solr-gsc SERVER_MACHINE_NAME /opt/drupal/solrconfig.zip
+unzip -d ~/solrconfig solrconfig.zip
+sudo cp ~/solrconfig/* /var/solr/data/SOLR_CORE/conf
+sudo systemctl restart solr
+```
+- `SERVER_MACHINE_NAME`: This should be the `machine_name` that was automatically generated when creating the configuration in the above step.
+
+### Adding an Index
+
+In order for content to be indexed back into Solr, a search index needs to be added to our server. Navigate to `/admin/config/search/search-api/add-index` and check off the things you'd like to be indexed.
+
+![Adding a Search Index](../../assets/adding_a_search_index.png)
```

Comment:
At this point, the media module has not been enabled yet and doesn't show up in the list like it does in this screenshot.